publish-token/src/main/java/fi/thl/covid19/publishtoken/generation/v1/PublishTokenGenerationController.java

```diff
@@ -28,10 +31,12 @@ public PublishTokenGenerationController(PublishTokenService publishTokenService,
     }
 
     @PostMapping
-    public PublishToken generateToken(@RequestHeader(name = SERVICE_NAME_HEADER) String rawRequestService, @RequestBody PublishTokenGenerationRequest request) {
+    public PublishToken generateToken(@RequestHeader(name = SERVICE_NAME_HEADER) String rawRequestService,
+                                      @RequestHeader(name = VALIDATE_ONLY_HEADER) Optional<Boolean> validateOnly,
```

Comment:
This could be something like:

`@RequestHeader(name = VALIDATE_ONLY_HEADER, required = false) boolean validateOnly`

Reply:
Yeah, that's cleaner. There's a slight functional variation (if both are defined), but nothing relevant. I changed it to this.